### PR TITLE
Create regular.c

### LINKED ISSUE #1093: regular falsi in c
I would like to do that problem. gssoc19

### PR DESCRIPTION
**Pull request by gourabk121**
Fixes #1093 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- create regular falsi

#### Languages Used:
- c

#### Files Added:
- regular.c
> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Regular_falsi/regular.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Girlscript Summer Of Code
+// Regular falsi in c
+
+//In mathematics, the false position method or regula falsi is a very old method for solving an equation in one unknown,
+//that, in modified form, is still in use. In simple terms, the method is the trial and 
+//error technique of using test ("false") values for the variable and then adjusting the test value according
+//to the outcome. This is sometimes also referred to as "guess and check".
+//Versions of the method predate the advent of algebra and the use of equations.
+
+#include <stdio.h>
```
Comment (MastersAbh) [inline Regular_falsi/regular.c]: remove space before <

**Comment by MastersAbh** [comment]
Hi @gourabk121 , please do the required changes.

**Comment by marwahmanbir** [comment]
@gourabk121 are you working on this?

**Comment by marwahmanbir** [comment]
@gourabk121 are you working on this? Just a day left for GSSoC to conclude.

**Comment by gourabk121** [comment]
@marwahmanbir  I have changed

**Comment by gourabk121** [comment]
@MastersAbh  changed it

**Comment by MastersAbh** [comment]
@rishabhgarg25699 you can merge it. @marwahmanbir  have a look.

**Pull request merged**